core/src/main/java/com/netflix/conductor/core/execution/WorkflowExecutor.java

```diff
@@ -795,7 +1006,7 @@ public void setTaskDomains(List<Task> tasks, Workflow wf) {
 
             } else {
                 tasks.forEach(task -> {
-                    if (!WorkflowTask.Type.isSystemTask(task.getTaskType())) {
+                    if (!TaskType.isSystemTask(task.getTaskType())) {
```

Comment:
`task.getTaskType().isSystemTask()`, add `isSystemTask` method that returns the boolean attribute back if it doesn't exist yet.

Reply:
Resolving as this is same explanation and resolution as in https://github.com/Netflix/conductor/pull/716#discussion_r213918055